.github/workflows/ci.yml

```diff
@@ -9,7 +9,6 @@ jobs:
       - uses: Brightspace/third-party-actions@actions/checkout
       - uses: Brightspace/third-party-actions@actions/setup-node
         with:
-          node-version: '14'
```

Comment:
Know this is a draft/trying it out but just FYI you'll want `node-version-file: .nvmrc` to pick up the `nvmrc` file. This is working to pick up node 16 since the default on runners is now node 16

Reply:
Ah gotcha -- thanks!